contentcuration/contentcuration/frontend/channelEdit/components/StudioTree/StudioTree.vue

```diff
@@ -76,23 +76,14 @@
                     <VFlex
                       class="px-1 caption text-truncate"
                       :class="getTitleClass(node)"
+                      grow
                     >
-                      <VTooltip
```

Comment:
Seems reasonable to remove the tooltip, as it was just re-displaying the title, yes?

Reply:
Yes, but it might just get annoying for reading longer titles that have been truncated. Hopefully it's enough to have the tooltip on the icon though

Reply:
Ah, right - maybe we can use the text truncator from KDS to handle this in the future (and maybe it will have better luck with its tooltops!)